README.rst

```diff
@@ -20,33 +20,85 @@ configurations remain synchronized with the state of all databases in the
 cluster. PgQ is used for buffering mutation records until they are collected by
 the relay and forwarded to their final destination.
 
+Usage
+=====
+
+PGShovel is a tool which replicates mutations (``INSERT``, ``UPDATE`` and ``DELETE`` operations) made to a PostgreSQL data out of the database and into an ordered stream of transaction-aware mutation messages.  These messages can then be consumed by 3rd party systems as needed.  Commonly, the mutation stream is consumed by data warehouse operations, cache updating services, search index services or can even be processed into other new streams.
+
+At Disqus PGShovel is actively being developed to replicate changes from our master Postgres database - which contains the core commenting and other relational data - into our data warehousing system. There is further planned work to also consume the mutation stream which is models mutations to rows and columns into a new stream of **object snapshots**.  Unlike mutations, these snapshots accurately and succinctly model our business objects and are free from the idiosyncrasies of attempting to read and interpret their serialized structure in the rows and columns of an RDBMS.
```

Comment:
>  There is further planned work to also consume the mutation stream which is models mutations to rows and columns into a new stream of **object snapshots**.  

Having a hard time parsing that... is this an accurate rephrasing?

>  There is further planned work to produce a new stream of **object snapshots** by consuming the stream of row and column model mutations.

I think the word `model` might be unnecessary here, but not sure